cloudify_cli/commands/deployments.py

```diff
@@ -227,10 +231,16 @@ def manager_create(blueprint_id,
             skip_plugins_validation=skip_plugins_validation
         )
     except MissingRequiredDeploymentInputError as e:
-        logger.info('Unable to create deployment. Not all '
-                    'required inputs have been specified...')
-        _print_deployment_inputs(client, blueprint_id)
-        raise CloudifyCliError(str(e))
+        expected_inputs_def = _get_deployment_inputs(client, blueprint_id)
```

Comment:
TBH, I don't much like that there are calculations inside an `except` clause. If something isn't right there, the error won't be caught, and will be even less helpful.

Reply:
@mcouthon I don't like it either, but this is the existing situation too. As of 4.1.1, we make a call to the manager to retrieve the blueprint in order to print the deployment inputs.